src/core/lib/experiments/config.cc

```diff
@@ -50,13 +50,21 @@ ForcedExperiment g_forced_experiments[kNumExperiments];
 
 std::atomic<bool> g_loaded(false);
 
+absl::AnyInvocable<bool(struct ExperimentMetadata)> g_check_constraints_cb =
```

Comment:
`AnyInvocable` is not trivially destructable, so we'll need to choose something else.
Probably in this case a pointer would be fine (we could even delete it once we've checked the constraints!)... so `absl::AnyInvocable<bool(struct ExperimentMetadata)>* g_check_constraints_cb`?

Reply:
Fixed.